src/core/SampleBuffer.cpp

```diff
@@ -505,76 +503,80 @@ f_cnt_t SampleBuffer::decodeSampleOGGVorbis( QString _f,
 
 	f_cnt_t frames = 0;
 
-	QFile * f = new QFile( _f );
-	if( f->open( QFile::ReadOnly ) == false )
+	QFile * f = new QFile(fileName);
+	if (f->open(QFile::ReadOnly) == false)
 	{
 		delete f;
 		return 0;
 	}
 
-	int err = ov_open_callbacks( f, &vf, NULL, 0, callbacks );
+	int err = ov_open_callbacks(f, &vf, nullptr, 0, callbacks);
 
-	if( err < 0 )
+	if (err < 0)
 	{
-		switch( err )
+		switch (err)
 		{
 			case OV_EREAD:
-				printf( "SampleBuffer::decodeSampleOGGVorbis():"
-						" media read error\n" );
+				printf("SampleBuffer::decodeSampleOGGVorbis():"
+						" media read error\n");
 				break;
 			case OV_ENOTVORBIS:
-/*				printf( "SampleBuffer::decodeSampleOGGVorbis():"
```

Comment:
I'm OK with commenting this out, but I think you should add a comment above describing *why* you commented it out.

Reply:
That's the thing, I don't know why it was commented out in the first place. It's a debug message when the OGG library returns this error while loading the sample, I don't know if it was commented out because it was triggered too often before or something. Tbh, I think it should be the other way around and there should be a comment explaining why the comment blocks were added :laughing: 

I could reverse that change if it's preferred though. Btw, I think this goes beyond the scope of code style formatting, but probably those should all be replaced by `qWarning`s

Reply:
Ah, now I see it! I thought you added the comments, but you removed them. The adding was done by @tobydox in 2006, so whatever it was may not be relevant anymore. I'll resolve this comment.

About `qWarning`, this can be done in another PR. But we have requests for a new logger class, so `qWarning` may not be correct.